sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -31,6 +31,86 @@ def create_endpoints(plugin):
     # stock tickers are 1-5 capital characters, with a dot allowed. To keep
     # this from triggering with random text we require a leading '$'
     plugin.on_message("s\$[A-Z\.]{1,5}", stock_quote, wait=False)
+    plugin.on_message("c\$[A-Z\.]{1,5}", crypto_quote, wait=False)
+
+async def crypto_quote(message, app):
+    match = STOCK_REGEX.search(message.get("text", ""))
+    if not match:
+        return
+
+    symbol = match.group("symbol")
+    LOG.debug("Fetching crypto quotes for symbol %s", symbol)
+
+    response = message.response()
+    try:
+        stocks = app["plugins"]["stocks"]
+        crypto = (await stocks.crypto())
+        quote = ''
+        for c in crypto:
+            if c["symbol"] == f"{symbol}USDT":
+                quote = c
+        if quote == '':
+            raise Exception
+        LOG.debug("Crypto quote from IEX API: %s", quote)
+    except Exception as e:
```

Comment:
Too broad and we risk missing valid `exception`. It would be best to only catch `ClientResponseError` and respond that there is an API problem.